File: Python Exams/01. Basic Exam Collection/Basics Online Exam - 18 and 19 July 2020/02. Add Bags/02. Add Bags.py
price, kg, days, luggage_count = [float(input()) for _ in range(4)]

if kg < 10:
    price *= 0.2
elif 10 <= kg <= 20:
    price *= 0.5

if int(days) < 7:
    price *= 1.4
elif 7 <= int(days) <= 30:
    price *= 1.15
else:
    price *= 1.1

print(f'The total price of bags is: {price * luggage_count:.2f} lv.')

'''
30
18
15
2
'''

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 18 and 19 July 2020/02. Add Bags/02. Add Bags.js
function addBags([price, kg, days, luggageCount]) {
    if (kg < 10) price *= 0.2
    else if (10 <= kg && kg <= 20) price *= 0.5

    if (days < 7) price *= 1.4
    else if (7 <= days && days <= 30) price *= 1.15
    else price *= 1.1

    return `The total price of bags is: ${(price * luggageCount).toFixed(2)} lv.`
}

// console.log(addBags(["30", "18", "15", "2"]))